README.md

```diff
@@ -16,13 +16,73 @@ To use it, extract the binary for your platform into the same folder as your `.t
 
 ## Configure the Provider
 
+### Default Space
+
+```hcl
+# main.tf
+
+provider "octopusdeploy" {
+  address = "http://octopus.production.yolo"
+  apikey  = "API-XXXXXXXXXXXXX"
+}
+```
+
+### Scoped to a single Space
+
+```hcl
+# main.tf
+
+provider "octopusdeploy" {
+  address = "http://octopus.production.yolo"
+  apikey  = "API-XXXXXXXXXXXXX"
+  space   = "Space-1"
+}
+```
+
+### Multiple spaces
+
+To manage resources in multiple spaces you currently must use multiple instances of the provider with [aliases](https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-instances) like so:
+
 ```hcl
 # main.tf
 
 provider "octopusdeploy" {
   address = "http://octopus.production.yolo"
   apikey  = "API-XXXXXXXXXXXXX"
 }
+
+provider "octopusdeploy" {
+  alias   = "space_1"
+
+  address = "http://octopus.production.yolo"
+  apikey  = "API-XXXXXXXXXXXXX"
+  space   = "Space-1"
```

Comment:
`Space-1` looks very similar to  Space Ids (e.g. `Spaces-1`) but it actually Space Name. This might confuse some users. I would use some other names here. E.g. `support`, `product1`.